app/assets/javascripts/components/reorderable-list.js

```diff
@@ -7,17 +7,27 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
   ReorderableList.prototype.start = function ($module) {
     this.$module = $module[0]
+    this.$upButtons = this.$module.querySelectorAll('.app-c-reorderable-list__actions button:first-of-type')
```

Comment:
I would've normally used a `.js-reorderable-list-up-button` class for querying the elements via JavaScript (as we do in all components) but components from the library don't allow custom classes to be added.

Reply:
In this specific case, it won't work as the `:first-of-type` pseudo selector only works on different element types. What would be the advantage of using the `.gem-c-button` class instead?

Sure, if we're not in a rush with this will start again the conversation around allowing classes to be passed in for components.

Reply:
Opened https://github.com/alphagov/govuk_publishing_components/pull/1344 for this.